test/DynamoCoreWpfTests/PythonNodeCustomizationTests.cs

```diff
@@ -145,16 +147,15 @@ public void OnTextAreaTextEnteringEventTest()
             codeEditor.Focus();
 
             //This will generate the event of going to the end of the line
-            var textArea = Keyboard.FocusedElement;
-            textArea.RaiseEvent(new KeyEventArgs(
-                Keyboard.PrimaryDevice,
-                PresentationSource.FromVisual(codeEditor),
-                0,
-                Key.End)
-            {
-                RoutedEvent = Keyboard.KeyDownEvent
-            }
-            );
+            var textArea = scriptWindow.editText.TextArea;
```

Comment:
is there any possibility that this any of these are null? Can we use `?` operator?

Reply:
This is a test. Do you still feel that some defensive coding is necessary here?

Reply:
For some reason, the previous logic of raising the event started failing with the new package. I couldn't figure out why so I used this other way of moving the cursor position to the end of the line for the purpose of this test.